static/js/tools/map/chart.tsx

```diff
@@ -89,10 +89,11 @@ export function Chart(props: ChartProps): JSX.Element {
   const statVarInfo = props.statVar.value;
   const [errorMessage, setErrorMessage] = useState("");
   const [denomInput, setDenomInput] = useState(props.statVar.value.denom);
-  const title = getTitle(
+  const chart_title=   getTitle(
     Array.from(props.dates),
     statVarInfo.info.title ? statVarInfo.info.title : statVarInfo.dcid
   );
+  const title = props.statVar.value.perCapita? 'Per Capita ' + chart_title : chart_title; 
```

Comment:
In Scatter Tool, we append "Per Capita" to the end of the stat var name. To keep things consistent, let's move "Per Capita" to the end of the chart title?

Reply:
Thank you for the review. I have moved 'Per Capita' to the end of the StatVar's name.